SPECS/cyrus-sasl/cyrus-sasl.spec

```diff
@@ -2,23 +2,26 @@ Summary:        Cyrus Simple Authentication Service Layer (SASL) library
 Name:           cyrus-sasl
 Version:        2.1.27
 Release:        4%{?dist}
+# License is a BSD with advertising variant 
```

Comment:
Thanks for checking the license. As we try to follow [Fedora's notation for licenses](https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing#License_of_Fedora_SPEC_Files), let's remove the comment and change the `License` tag to `License: BSD with advertising`. Unless the license you've seen doesn't really resable the [BSD with advertising one from Fedora's example](https://fedoraproject.org/wiki/Licensing:BSD?rd=Licensing/BSD#BSDwithAdvertising).

Reply:
Thanks for addressing this. I initially  assumed there was a reason as to why this was here in the first place, but the license looks pretty close to the OG 4-clause BSD that you linked. Will fix!